### PR TITLE
Made testing build with release settings

### PR DESCRIPTION
**Pull request by AndriiHorishniiMOC**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-2444

**What was solved?**

- Added ability to make local maven release.
- Set "minifyEnabled true" for the test application build that is used for the acceptance testing.
- Set "release" as a build variant for the testing app build task.

**Release notes:**

 - [ ] Feature
 - [x] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [ ] Tests updated? Added? Unit, acceptance, snapshots?
 - [ ] Logging for future troubleshooting of client issues added?

**Screenshots:**

### CODE REVIEW

**Review by DavDo — APPROVED** [review]
LGTM

**Review by andrews-moc — APPROVED** [review]

File: bitrise.yml [hunk 1]
```diff
@@ -412,7 +412,10 @@ app:
     ANDROID_CORTEX_BANKING_APP_SLUG: 0376f45568cb22dd
   - opts:
       is_expand: false
-    INTEGRATOR_VARIANT: debug
+    DEBUG_VARIANT: debug
+  - opts:
+      is_expand: false
+    INTEGRATOR_VARIANT: release
```
Comment (gugalo) [inline bitrise.yml:418]: Why not just 'RELEASE_VARIANT'?

**Review by gugalo — APPROVED** [review]

**Review by gugalo — CHANGES_REQUESTED** [review]
Could we discuss this on stendup on Monday

File: build.gradle [hunk 2]
```diff
@@ -27,6 +27,9 @@ allprojects {
       dirs '../libs'
     }
 
+    // Needed for some automated and manual testing (e.g: acceptance tests)
+    mavenLocal()
+
     // Internal maven repository with Core SDK build from `feature/live-observation` branch
     // TODO: delete before merging to `development` branch
     maven { url "https://s01.oss.sonatype.org/content/repositories/comglia-1253" }
```
Comment (DavDo) [inline build.gradle:35]: Could you please remove this since you have touched this file)

**Review by DavDo — APPROVED** [review]
LGTM

**Review by Pelkar — APPROVED** [review]
LGTM

**Review by andrews-moc — APPROVED** [review]

**Comment by AndriiHorishniiMOC** [comment]
!merge

**Comment by AndriiHorishniiMOC** [comment]
!squash

**Pull request merged**